Source/Scene/Cesium3DTileset.js

```diff
@@ -984,51 +988,60 @@ define([
                 } else {
                     // Tile does not meet SSE.
 
+                    // Get visibility for all children. Check if any visible children are not loaded.
+                    var allVisibleChildrenLoaded = true;
+                    var someVisibleChildrenLoaded = false;
+                    for (k = 0; k < childrenLength; ++k) {
+                        child = children[k];
+                        child.parentPlaneMask = planeMask;
+                        child.planeMask = child.visibility(frameState.cullingVolume);
+                        // If the child is visible...
+                        if (child.planeMask !== CullingVolume.MASK_OUTSIDE) {
+                            if (child.contentReady) {
+                                someVisibleChildrenLoaded = true;
+                            } else {
+                                allVisibleChildrenLoaded = false;
+                            }
+                        }
+                    }
+
                     // Only sort children by distance if we are going to refine to them
                     // or slots are available to request them.  If we are just rendering the
                     // tile (and can't make child requests because no slots are available)
                     // then the children do not need to be sorted.
 
-                    var allChildrenLoaded = t.numberOfChildrenWithoutContent === 0;
-                    if (allChildrenLoaded || t.canRequestContent()) {
+                    if (someVisibleChildrenLoaded || t.canRequestContent()) {
                         // Distance is used for sorting now and for computing SSE when the tile comes off the stack.
                         computeDistanceToCamera(children, frameState);
 
                         // Sort children by distance for (1) request ordering, and (2) early-z
                         children.sort(sortChildrenByDistanceToCamera);
-// TODO: same TODO as above.
                     }
 
-                    if (!allChildrenLoaded) {
-                        // Tile does not meet SSE.  Add its commands since it is the best we have and request its children.
+                    if (!allVisibleChildrenLoaded) {
```

Comment:
Is there a comment here for the case empty parent case that we discussed offline today?